### DISCUSSION TITLE
Fitting `pyddm` models without any conditions

### DISCUSSION THREAD

**Discussion by conorheins** [mwshinn/PyDDM#70] [Ask for help] [answered]
Hi pyddm community + developers,

First of all, thank you for developing this great package! I've just started using it / getting familiarized with the package structure.

I have a question about generating `Sample` objects from e.g. `numpy.ndarray` or `pandas.DataFrame` data structures, in the case when there are no experimental conditions at play (e.g. no coherence level or subject ID). In the `pyddm` [Quickstart Guide](https://pyddm.readthedocs.io/en/stable/quickstart.html), the first example involves creating a`Sample` object when there is no experimental condition at play (from what I understand, at least). However, that example is not created from data, but by sampling from a `solution` object directly. Therefore, I am wondering how to create a `Sample` object from data but _without_ conditions, using the function `Sample.from_numpy_array(M, conditions)`.  In this case, the data array `M` just has two columns, `M[:,0]` stores reaction times, and `M[:,1]` stores the correct/incorrect Boolean associated with the choice for that trial.

In the case of no conditions, would the ideal usage be to just pass in the `conditions` argument as an empty list, e.g. `Sample.from_numpy_array(M, conditions = [])`? I have been doing that and it seems to work, but I just want to make sure that's the correct usage and I'm not inadvertently creating issues with my model fitting in the case of no conditions.

Best wishes,
Conor

**Comment by mwshinn** [comment, marked as answer]
Yes, you're correct.  If you want to confirm, for a sample "samp", you can always do "samp.to_pandas_dataframe()" [(docs here)](https://pyddm.readthedocs.io/en/latest/apidoc/model.html#pyddm.sample.Sample.to_pandas_dataframe) and look at the output.
